pkg/management/authorization.go

```diff
@@ -91,6 +91,7 @@ var (
 	MGMTGetRealmBackOfficeConfiguration     = newAction("MGMT_GetRealmBackOfficeConfiguration", security.ScopeGroup)
 	MGMTUpdateRealmBackOfficeConfiguration  = newAction("MGMT_UpdateRealmBackOfficeConfiguration", security.ScopeGroup)
 	MGMTGetUserRealmBackOfficeConfiguration = newAction("MGMT_GetUserRealmBackOfficeConfiguration", security.ScopeRealm)
+	MGMTGetFederatedIdentities              = newAction("MGMT_GetFederatedIdentities", security.ScopeRealm)
```

Comment:
Do not forget to notify new authorization to the front end !

Reply:
It should be ScopeGroup as we know the target user and we use CheckAuthorizationOnTargetUser